var/spack/repos/builtin/packages/py-hepstats/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHepstats(PythonPackage):
+    """hepstats is a library for statistical inference aiming
+    to cover the needs in High Energy Physics.
+    It is part of the Scikit-HEP project.
+    """
+
+    homepage = "https://github.com/scikit-hep/hepstats"
+    pypi = "hepstats/hepstats-0.8.1.tar.gz"
+    maintainers("jonas-eschle")
+
+    license("BSD-3-Clause", checked_by="jonas-eschle")
+
+    tags = ["likelihood", "statistics", "inference", "fitting", "hep"]
+
+    version("0.9.2", sha256="cf929871d45e338492eef585faaaa23eff93b200b4787d6b6181dc81f2607be7")
+    version("0.8.1", sha256="ebb890496d7aebbf1d717de15d073be31d6775065308a4e0f263ed4051992b3f")
+
+    depends_on("python@3.9:", type=("build", "run"), when="@0.8:")
+    depends_on("py-setuptools@42:", type="build")
+    depends_on("py-setuptools-scm@3.4:+toml", type="build")
```

Comment:
Looks like the dependency on `py-setuptools*` changed for the newer version:

```
[build-system]
requires = ["hatchling", "hatch-vcs"]
...
```

Reply:
You're right! Just caught it when I tried to build. Fixed